docs/starters.yml

```
@@ -6781,3 +6781,19 @@
     - Landing Page
     - Google Analytics
     - PaperCSS style
+- url: https://mui-treasury.com/layout/clones/reactjs?bgColor=b6c0d4
+  repo: https://github.com/mui-treasury/gatsby-mui-layout-starter
+  description: Supercharge your next project with Mui Treasury Layout, built on top of Material-UI
+  tags:
+    - Layout
+    - Styling:CSS-in-JS
+    - Styling:Material-UI
```

```suggestion
    - Styling:Material
```

This one needs to match the tag in the allow list exactly.